CHANGELOG.md

```diff
@@ -1,14 +1,23 @@
 # Change Log for hipBLAS
 
-## (Unreleased) hipBLAS
+## (Unreleased) hipBLAS 0.50
 ### Added
 - Added library version and device information to hipblas-test output.
+- Added --rocsolver-path command line option to choose path to pre-built rocSOLVER, as
+  absolute or relative path.
+- Added --cmake_install command line option to update cmake to minimum version if required
+- Added cmake-arg parameter to pass in cmake arguments while building
 
 ### Fixed
 - Added hipblasVersionMinor define. hipblaseVersionMinor remains defined
   for backwards compatibility.
 
-## [hipBLAS 0.49.0 for ROCm 5.0.0]
+### Changed
+- rocblas-path command line option can specify either absoulute or relative path now
```

Comment:
Spelling error on absolute.
I'd also change "can specify" to "can be specified as"